deploy/cloudflare.md

```diff
@@ -11,29 +11,38 @@ outline: deep
 
 Uploading via Web UI is limited to 300 MB.
 
-Use [rclone](https://rclone.org/downloads/) to upload larger PMTiles archives to R2.
+Use [rclone](https://rclone.org/downloads/) to upload larger PMTiles archives to R2 (you can use the `rclone/rclone` docker image in order to avoid installation, the config is at `/etc/rclone` for mounting). 
+In order to use rclone you'll need to create an API key. Copy the "Access Key ID", the "Secret Access Key" and the "Endpoints for S3 clients" from the API key creation screen.
+Run the following commands:
+1. `rclone config` and follow the on screen questions, note that the endpoint should be `https://<ACCOUNT_ID>.r2.cloudflarestorage.com/<BUCKET_NAME>`
+2. `rclone copy your.pmtiles <the name you gave the above rclone configuration):<FOLDER_INSIDE_THE_BEUCKET> to upload the file
 
 Name your uploads to storage with the `.pmtiles` extension. Your tile requests to the Workers URL will look like `/FILENAME/0/0/0.ext` for the archive `FILENAME.pmtiles`.
 
 ### 2. Create Worker with Web Console
 
-1. In the Workers tab of the Cloudflare dashboard, choose **Create a Service**.
+1. In the Workers left menu of the Cloudflare dashboard, choose **Create Worker**.
 
-2. Leave the default **HTTP handler** option.
+2. It will ask you to deply it first before you can edit the code, click **Deploy**
 
-3. in Settings for your worker, choose Variables > R2 Bucket Bindings > **Add Binding**.
+3. Paste in this code from: [index.js](https://protomaps.github.io/PMTiles/index.js).
 
-  * Create variable with name `BUCKET` and select your R2 bucket from Step 1.
+4. Leave the default **HTTP handler** option.
 
-  * Choose **Save and Deploy**.
+5. Choose **Save and Deploy** and leave the code editing window.
+  
+6. Select the newly create worker from the workers list
 
-4. in **Quick Edit**, paste in this code: [index.js](https://protomaps.github.io/PMTiles/index.js).
+7. In Settings of your worker, choose Variables > R2 Bucket Bindings > **Add Binding**.
+
+  * Create variable with name `BUCKET` and select your R2 bucket from Step 1.
 
   * Choose **Save and Deploy**.
 
+
 Your worker should now be active at its `*.workers.dev` domain. 
 
-Make a request for `/FILENAME/0/0/0.EXT` to verify tiles are served.
+Make a request for `<ACCOUNT_NAME>.workers.dev/<FOLDER>/<FILENAME>/0/0/0.pbf` to verify tiles are served.
```

Comment:
`<FOLDER>` should not be necessary if the archives are in the root of the bucket

Reply:
change `.pbf` to `.mvt` as that is the preferred extension going forward. I guess people could have went through this setup for a non-vector tile source, maybe make a note of that (it should also give a helpful error message if you ask for the wrong format)

Reply:
I was not able to upload to the root of the bucket using the instruction here.
So I preferred correct instructions over stuff that doesn't work but look nice :-)

Reply:
```suggestion
Make a request for `<ACCOUNT_NAME>.workers.dev/<FOLDER>/<FILENAME>/0/0/0.<mvt | pbf | png>` to verify tiles are served.
```

Reply:
We actually want to deprecate `pbf` in the near future because it is cached separately than `mvt` so maybe leave it off

Reply:
Removed.